user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/Ad_hoc_data/Ad_hoc_Tutorials_Satellites.md

```diff
@@ -0,0 +1,89 @@
+---
+uid: Ad_hoc_Tutorials_Satellites
+---
+
+# Building a GQI data source that fetches satellites
+
+In this tutorial, you will learn how you can create a GQI data source that can be used to add ad hoc data to a dashboard or low-code app.
+
+Expected duration: 15 minutes.
+
+## Prerequisites
+
+- [Visual Studio](https://visualstudio.microsoft.com/downloads/) with [DataMiner Integration Studio](xref:Installing_and_configuring_DataMiner_Integration_Studio)
```

Comment:
Should a specific version of Visual Studio be used (e.g. 2022)? If yes, could you clarify this?

Reply:
Is not necessary here